server/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+const PORT = 3001;
+const House = require('../db/Schemas/Listing.js');
+
+
+app.use(express.json());
+
+
+app.route('/api/listings')
+    .get((req, res) => {
+        console.log(req.url);
+        // console.log(House);
+        // res.sendStatus(200);
```

Comment:
Would be good to get rid of console.logs and unnecessary comments.

Reply:
Oh yeah, that's leftover from my postman tests